### PR TITLE
feat: refactor task profiling fix

### PR DESCRIPTION
**Pull request by guilhem-barthes**
## Description

Our previous fix for `task_profiling` when it already existed (https://github.com/Substra/substra-backend/pull/559) had a flaw: as we override the `create` in the serializer to update when existing, we broke the REST status code (always returning `201` even when updating). This PR changes the behavior, and introduces a new error, `OrganizationHttpError` which contains information from `requests.exceptions.HttpError` but still child of `OrganizationError`.

## Checklist

- [x] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203663111447729

### CODE REVIEW

**Review by SdgJlbl — APPROVED** [review]
Thanks 👍

File: backend/api/models/task_profiling.py [hunk 1]
```diff
@@ -21,7 +22,7 @@ class TaskProfiling(models.Model):
     compute_task = models.OneToOneField(
         "ComputeTask", primary_key=True, on_delete=models.CASCADE, related_name="task_profiling"
     )
-    creation_date = models.DateTimeField(auto_now_add=True)
+    creation_date = models.DateTimeField(default=timezone.now)
```
[the remark this answers is not in the document]
Comment (guilhem-barthes) [reply backend/api/models/task_profiling.py:25]: I can check again, but from memory, when setting _auto_now_add, I add trouble changing it manually. But I made some changes after, I would indeed check if this change is still needed

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Pull request merged**